### ISSUE TITLE
[Bug Report][3.1.6] v-select "header", "divider", and "disabled" options not working for array of objects passed to "items" prop

### ISSUE THREAD

**Issue by rjskene** [vuetifyjs/vuetify#16792]
### Environment
**Vuetify Version:** 3.1.6
**Vue Version:** 3.3.0-alpha.4
**Browsers:** Chrome 110.0.0.0
**OS:** Mac OSX

### Steps to reproduce
Just provide items prop with array of objects where one object has only the "header" key set to some string value.

### Expected Behavior
Should create an item in a header-style that cannot be selected.

### Actual Behavior
fails

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqVUUFOwzAQ/IrlS0BqbEoRh6hF5cYDuDU9hGRLTWPHsjdFKMrfWTsplLYgcUp2Z3dmdrzq+KO1Yt8Cz/gcQdu6QHjIDWPzfVpYG39joQtlxirWHmoo8dBgjCaaCupFzhE85vyA5JgpIvYExO83Ir/oui7s9P0oJo/UQjH4mMsjf1T60imLzAO2AVbaNg5ZxxxsWM82rtEsocOSMFw2xiMLImwRBq5MW9fXh370RcAqCHZbKCpwGUuew/hTrJJ+EjFUWANBkWl6sXub0BnrYHcwSNb4hA/u6Cwr3nxjKO0urFImEaBUMhY7oUeuQ53zLaL1mZStsbtXUTZaLgmTrjWoNKRVo5czMRM3aVHbbSHuZKU8HsMCvE5fXPPuwZFwzqPhUQPV5uM3nREm+qm4H2jHVqA8o5LEvweXOjCUFrg/7Z/MXjrhZOTsjCBNMfeUbK3Mzp+EWvoYaHzPUP/rPq2MiARhmx6y5/2E//BIqnGPrz8ByLQStg==)

**Comment by hbgl** [comment]
Same is true for `v-autocomplete` and `v-combobox`.

**Comment by ShoryuKyzan** [comment]
For VSelect:
Reason this happens is VSelect overrides default slot of VList I think. VList implements these properties (VListChildren). (this is from memory, encountered this a while ago).
TLDR unimplemented.
 
Workaround is to render the item yourself depending on item.raw data that can be passed in.
For options api just use the template and make sure items have the appropriate values specified on the item object.

```
<template>
  <v-select
    label="Label"
    :modelValue="selected"
    :items="list"
    @update:modelValue="selected = $event"
  >
    <template v-slot:item="{ item, props }">
      <v-divider v-if="item.raw.type === 'divider'" />
      <v-list-subheader v-else-if="item.raw.type === 'subheader'">{{
        item.title
      }}</v-list-subheader>
      <v-list-item v-else v-bind="props" :disabled="item.raw.disabled" />
    </template>
  </v-select>
</template>
<script setup lang="ts">
import { ref, watch } from "vue";

interface ListItem {
  title?: string;
  value?: string;
  type?: "divider" | "subheader";
  disabled?: boolean;
}
const list: ListItem[] = [
  { type: "divider" },
  { title: "SubHeader", type: "subheader" },
  { title: "foo", value: "foo" },
  { title: "disabled-arb", value: "arb", disabled: true },
  { title: "bar", value: "bar" }
];
const selected = ref({ title: "bar", value: "bar" });

watch(
  () => selected.value,
  val => console.log("new value", val)
);
</script>
```

![image](https://user-images.githubusercontent.com/32823296/224869471-1f1b9f3c-ab3b-41ed-8fec-14c050c02709.png)

**Comment by 1gn0r4nd** [comment]
Try the following:
```
<template>
  <v-app>
    <v-main>
      <v-select
        v-model="test"
      	:items="items"
        label="Test Header"
      />
      {{test}}
    </v-main>
  </v-app>
</template>

<script setup>
import { ref } from 'vue'

const test = ref(null)
const items = [
  {title: 'test 1'},
  {title: 'test 2'}
]
</script>

```

**Comment by 1gn0r4nd** [comment]
Your reproduction link doesn't work. In the API I only see disabled for the select, not per item (I added a disabled: true to the item, but it has no effect, as there is no code written to make it not selectable Guessing that would be a feature request).
Try the following code:
```
<template>
  <v-app>
    <v-main>
      <v-select
        v-model="test"
      	:items="items"
        label="Test Header"
      />
      {{test}}
    </v-main>
  </v-app>
</template>

<script setup>
import { ref } from 'vue'

const test = ref(null)
const items = [
  {title: 'test 1'},
  {title: 'test 2', disabled: true}
]
</script>

```

**Comment by KaelWD** [comment]
Duplicate of #15721

**Comment by toto922** [comment]
```
<template>
  <v-app>
    <v-main>
      <v-select
        v-model="test"
      	:items="items"
        label="Test Header"
      />
      {{test}}
    </v-main>
  </v-app>
</template>

<script setup>
import { ref } from 'vue'

const test = ref(null)
const items = [
  {title: 'test 1'},
  {title: 'test 2', props: { disabled: true }}
]
</script>
```
